maps-compose-utils/src/main/java/com/google/maps/android/compose/kml/parser/KmlStyleParser.kt

```diff
@@ -167,6 +206,18 @@ internal class KmlStyleParser {
             }
         }
 
+        private fun convertStringToColor(color: String): ComposeColor {
+            require(color.length == 6 || color.length == 8)
+            val colorLong = color.toLong(radix = 16)
+            val colorWithAlpha = if (color.length == 6) {
+                0xFF000000.toInt() or colorLong.toInt()
```

Comment:
als color.length 6 is. krijg je dan niet altijd 0xFF000000? of werkt die or als een soort alpha layer.